Localization: big docs update (#8961)
DevExpress/devextreme-documentation: concepts/05 UI Components/PolarChart/10 Visual Elements/070 Axis Labels.md

Reviewer: ```suggestion
Axis labels are generated automatically. However, if needed, you can change their settings. Label settings are set using the [label](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/commonAxisSettings/label '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/commonAxisSettings/label/') object of the **argumentAxis**, **valueAxis**, or **commonAxisSettings** configuration object.
```

To specify the required font settings for label text, specify properties of the **label**.[font](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/commonAxisSettings/label/font '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/commonAxisSettings/label/font/') object.

* **Text Customization and Formatting**			

> You can change the way the default text is displayed by labels. For this purpose, use the [format](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/argumentAxis/label/format.md '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/argumentAxis/label/#format')  label property. If it is required, you can set a custom text to be shown by labels. For this purpose, implement a function that returns the required text and assign it to the [customizeText](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/argumentAxis/label/customizeText.md '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/argumentAxis/label/#customizeText') property of the **label** object. For details on data formatting, refer to the [Value Formatting](/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting '/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting/00%20Value%20Formatting.md') topic.

Reviewer: ```suggestion
You can change the way default text is displayed by labels. To do so, use the [format](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/argumentAxis/label/format.md '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/argumentAxis/label/#format')  label property. If required, you can set a custom text to be shown by labels: implement a function that returns the required text and assign it to the [customizeText](/api-reference/10%20UI%20Components/dxPolarChart/1%20Configuration/argumentAxis/label/customizeText.md '/Documentation/ApiReference/UI_Components/dxPolarChart/Configuration/argumentAxis/label/#customizeText') property of the **label** object. For details on data formatting, refer to the [Value Formatting](/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting '/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting/00%20Value%20Formatting.md') topic.
```
